src/Sylius/Bundle/CoreBundle/SyliusCoreBundle.php

```diff
@@ -47,6 +48,7 @@ public function build(ContainerBuilder $container)
         $container->addCompilerPass(new LazyCacheWarmupPass());
         $container->addCompilerPass(new SitemapProviderPass());
         $container->addCompilerPass(new RegisterTaxCalculationStrategiesPass());
+        $container->addCompilerPass(new RegisterOrderProcessorsPass());
```

Comment:
Then, please update Sylius to the newest master, because in #5971 I have moved it to Order component :)